app/models/price_policy.rb

```diff
@@ -17,7 +17,7 @@ class PricePolicy < ApplicationRecord
 
   with_options if: -> { SettingsHelper.feature_on?(:price_policy_requires_note) } do
     # Length of 10 is defined by Dartmouth.
-    validates :note, presence: true, length: { minimum: 10, allow_blank: true }
+    validates :note, presence: true, length: { minimum: 10, allow_blank: true }, on: :create
```

Comment:
I brought this exact case to Kayla in the ticket, and she said to proceed with it, so I think that it’s fine and we shouldn’t police the users that much — ultimately, someone could enter `XXXXXXXXXX` on the creation to work around the restriction, but they’d incur the wrath of the finance team 😄

Reply:
I *also* agree that it’s weird to check `valid?` in the view, and would vote to get rid of it (while keeping this change, so that we don’t have invalid records in the database because of this validation).

Reply:
If Kayla is fine with that scenario, then I'm cool with it.

For testing the removal of the `valid?` check, I might recommend using a snapshot of NU, and finding some really old price policies and trying to view them. My guess is there are possibly some _old_ price policies that are technically invalid now.